### PR TITLE
[BrokenLinksH2] Fixed the link path

### PR DESCRIPTION
**Pull request by v-sidlama**
Fixed the link path

### CODE REVIEW

File: iis/overview/centralized-web-farm-management.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Web content can also be shared on a network volume across multiple servers. This
 
  ![Screenshot of the Install a Package window.](centralized-web-farm-management/_static/ms-deploy-big.png) 
 
-IIS supports the [Microsoft Web Deployment Tool](/iis/publish/using-web-deploy.md), which provides support for synchronizing:
+IIS supports the [Microsoft Web Deployment Tool](/iis/publish/using-web-deploy/use-the-web-deployment-tool), which provides support for synchronizing:
```
Comment (John-Hart) [inline iis/overview/centralized-web-farm-management.md:26]: It looks like use-webdeployement-tool is a folder of samples. The introduction maybe better: iis/publish/using-web-deploy/introduction-to-web-deploy.md

**Review by John-Hart — CHANGES_REQUESTED** [review]

File: iis/overview/centralized-web-farm-management.md (hunk 1 above)
Comment (v-sidlama) [reply iis/overview/centralized-web-farm-management.md]: Thank you for pointing out the better link. it's updated. Please review

File: iis/overview/centralized-web-farm-management.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ Web content can also be shared on a network volume across multiple servers. This
 
  ![Screenshot of the Install a Package window.](centralized-web-farm-management/_static/ms-deploy-big.png) 
 
-IIS supports the [Microsoft Web Deployment Tool](/iis/publish/using-web-deploy.md), which provides support for synchronizing:
+IIS supports the [Microsoft Web Deployment Tool](iis/publish/using-web-deploy/introduction-to-web-deploy), which provides support for synchronizing:
```
Comment (John-Hart) [inline iis/overview/centralized-web-farm-management.md:26]: I think you need  the file extension ".md"

**Review by v-sidlama — COMMENTED** [review]
updated as suggested

File: iis/overview/centralized-web-farm-management.md (hunk 2 above)
Comment (Rick-Anderson) [reply iis/overview/centralized-web-farm-management.md]: Yup
Line 26: [Warning] Invalid file link: 'iis/publish/using-web-deploy/introduction-to-web-deploy'.

Comment (v-sidlama) [reply iis/overview/centralized-web-farm-management.md]: Sorry about that. Just added the extension. Please review

**Comment by Rick-Anderson** [comment]
@v-sidlama  check the build report
`Line 26: [Warning] Invalid file link: 'iis/publish/using-web-deploy/introduction-to-web-deploy.md'.`

image

File: iis/overview/centralized-web-farm-management.md [hunk 3]
```diff
@@ -23,7 +23,7 @@ Web content can also be shared on a network volume across multiple servers. This
 
  ![Screenshot of the Install a Package window.](centralized-web-farm-management/_static/ms-deploy-big.png) 
 
-IIS supports the [Microsoft Web Deployment Tool](/iis/publish/using-web-deploy.md), which provides support for synchronizing:
+IIS supports the [Microsoft Web Deployment Tool](iis/publish/using-web-deploy/introduction-to-web-deploy.md), which provides support for synchronizing:
```
Comment (John-Hart) [inline iis/overview/centralized-web-farm-management.md:26]: Sorry I think the doc the one the original author meant to link to was:  ..\publish\using-web-deploy\using-web-deploy-for-web-farms.md

Notice, the current doc is under iis/overview, so you'll need to change the relative path

**Review by v-sidlama — COMMENTED** [review]
added slash

File: iis/overview/centralized-web-farm-management.md (hunk 2 above)
Comment (John-Hart) [reply iis/overview/centralized-web-farm-management.md]: it looks like the [Shared Configuration](https://github.com/MicrosoftDocs/iis-docs/blob/1940addb97393648b8a68de46a99c7518828b445/IIS/manage/managing-your-configuration-settings/shared-configuration_264) link is also broken

Comment (v-sidlama) [reply iis/overview/centralized-web-farm-management.md]: It's working on my end. This is the [source page](https://docs.microsoft.com/en-us/iis/overview/centralized-web-farm-management) I'm checking. Please confirm

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@v-sidlama this has been merged to live and should be visible in 15 minutes. Can you verify the live version?

**Comment by Rick-Anderson** [comment]
Can you do another PR and drop the .md?

**Comment by v-sidlama** [comment]
> Can you do another PR and drop the .md?

Just did. Please review

**Pull request merged**